angular.json

```diff
@@ -126,14 +132,8 @@
                 "lint": {
                     "builder": "@angular-devkit/build-angular:tslint",
                     "options": {
-                        "tsConfig": [
-                            "apps/docs/tsconfig.app.json",
-                            "apps/docs/tsconfig.spec.json"
-                        ],
-                        "exclude": [
-                            "**/node_modules/**",
-                            "!apps/docs/**"
-                        ]
+                        "tsConfig": ["apps/docs/tsconfig.app.json", "apps/docs/tsconfig.spec.json"],
```

Comment:
Why are you changing formating of these section that you havent touched ?

Reply:
I am not explicitly modifying formatting, I have prettier installed, every time I save the file, it changes this file's formatting :(

Reply:
ok if this prettier it should be ok